types/pkgcloud/README.md

```diff
@@ -0,0 +1,7 @@
+This file exists for reminding contributors that this definition is **WIP**.
```

Comment:
I don't think this README is totally necessary - people don't tend to look here anyway. Perhaps add some comments to the .d.ts file itself if the warning is important?

Also, _everything_ on DT is a work in progress 😃

Reply:
I searched the repo for "WIP" to see how others indicated an implementation was incomplete, and only saw one that did so and it was done with a README.

Reply:
No problem either way.